projects/ngx-joystick/src/lib/ngx-joystick.component.ts

```diff
@@ -32,23 +34,33 @@ export class NgxJoystickComponent implements OnInit, OnDestroy {
   @Output() plainRight = new EventEmitter<JoystickEvent>();
 
   manager: nipplejs.JoystickManager;
+  private interval: NodeJS.Timeout;
```

Comment:
The library wont build if we use `NodeJS.Timeout` type here since that is not available to the browser. We could make the type `any`, or below on line 43 when setting the interval, we can use `window.setInterval()` instead. That will show the return type is a number (the id of the interval) and then we can declare the type of `interval` to be `number`.

You can test if the library builds by running `ng build ngx-joystick` from the root of the repo. So you run `ng build ngx-joystick` and then run `ng serve` to see your changes to the library in action.

Reply:
Thanks. I've corrected it. 
From the name, `NodeJS.Timeout` definitely belongs to the 'node' backend. Not the browser.
But I don't my why in my project,  `NodeJS.Timeout` compiles with `ng build`.

Reply:
Nice thanks. And that's interesting you don't see it. This is what I see when the `NodeJS.Timeout` is used:
<img width="1044" alt="Screen Shot 2021-12-02 at 9 55 32 PM" src="https://user-images.githubusercontent.com/17417149/144537343-6cc49196-30ec-4cba-b44f-fcd285c1dd72.png">